tools/WinObjC.Tools/vsimporter-templates/WinStore10-Package/Package.targets

```diff
@@ -0,0 +1,14 @@
+﻿<?xml version="1.0" encoding="utf-8" standalone="no"?>
+
+<!--
+  PACKAGE TARGETS
+  Description: The target definitions specific to $projectname$
+  For Use By: A VcxProj that depends on the $projectname$ package
+-->
+
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildThisFileDirectory)\$(MSBuildThisFileName).Consumption.targets" Condition="Exists('$(MSBuildThisFileDirectory)\$(MSBuildThisFileName).Consumption.targets')" />
```

Comment:
Bringing in the package's Consumption.targets file, which should be right next to it in the package. Nuget does something similar but they had some special syntax to get the package's installed path.

---
In reply to: [100941133](https://github.com/Microsoft/WinObjC/pull/1958#discussion_r100941133) [](ancestors = 100941133)